### PR TITLE
closes #64 Add User view and reducer

### LINKED ISSUE #64: Frontend: All Users Skeleton

### PR DESCRIPTION
**Pull request by gabrielwr**
closes #64

### CODE REVIEW

**Comment by gabrielwr** [comment]
This is by no means done-- especially the later part of the reducer. Just want to add files now and update them as needed/as we go.

File: app/main.jsx [hunk 1]
```diff
@@ -32,21 +31,28 @@ const EmptyApp = connect(
...
-        <Route path="/products" onEnter={ allProductsOnEnter } component={ AllProducts } />
+      <Route path="/" component={ Root } onEnter={ onAppEnter }>
+        <Route path="/products" component={ AllProducts } />
         {/*products/add is an admin only view*/}
         <Route path="/products/add" component={ EmptyApp } />
         <Route path="/products/:id" component={ SingleProduct } />
-        <Route path="/users" component={ EmptyApp } />
+        <Route path="/users" component={ AllUsers } />
         <Route path="/users/:id" component={ EmptyApp } />
         <Route path="/account" component={ EmptyApp } />
```
Comment (lada-tech) [inline app/main.jsx]: You could add Single user as a component already

File: app/reducers/user.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+export const deleteUser = (userId) => ({
+  type: DELETE_USER,
+  userId: userId
+})
+
+export const addUserToOrder = (user) => ({
+  type: ADD_ORDER_TO_USER,
+  user: user
+})
+
```
Comment (lada-tech) [inline app/reducers/user.jsx:51]: or add order to user?

File: app/reducers/user.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+export const removeUserFromOrder = (user) => ({
+  type: REMOVE_ORDER_FROM_USER,
+  user: user
+})
```
Comment (lada-tech) [inline app/reducers/user.jsx]: same here, action type and const in conflict

**Review by lada-tech — APPROVED** [review]
I think merge it so we can continue

**Pull request merged**